index.d.ts

```diff
@@ -128,6 +133,14 @@ export const Either: TypeRep;
 export function Left<A, B>(value: A): Either<A, B>;
 export function Right<A, B>(value: B): Either<A, B>;
 
+//  Placeholder
+
+interface Placeholder {
+  '@@functional/placeholder': boolean
```

Comment:
For objects, TS always widen their type to non-literal, so in this case you have to assert it to `true` manually, e.g.

```ts
const _ = {'@@functional/placeholder': true as true}; //=> {'@@functional/placeholder': true}
```

```ts
const _ = {'@@functional/placeholder': true}; //=> {'@@functional/placeholder': boolean}
```